addons/rose/addon/styles/rose/components/form/_radio-card.scss

```diff
@@ -0,0 +1,219 @@
+@use '../../variables/sizing';
+@use '../../utilities/type';
+@use '../../variables/typography';
+
+$l: sizing.rems(l); // 16
+$m: sizing.rems(m); // 16
+$size-xs: sizing.rems(xs); // 8
+$size-xxs: sizing.rems(xxs); // 4
+$size-xxxs: sizing.rems(xxxs); // 2
+$size-xxxxs: sizing.rems(xxxxs); // 1
+
+.rose-form-radio-card-wide {
+  position: relative;
+  display: flex;
+  align-items: center;
+  width: ($m) * 23;
+  justify-content: center;
+
+  .radio-card-overlay {
+    position: absolute;
+    height: $l + $size-xxs;
+    background-color: var(--ui-gray-subtler-5);
+    width: 100%;
+    bottom: 0;
+    right: 0;
+  }
+  .rose-form-radio-label {
+    font-style: normal;
+    font-weight: bold;
+    font-size: typography.font-size(s);
+    color: var(--black);
+    width: ($m) * 23;
```

Comment:
- Avoid hardcoding a measurement number (use sizing).
- Avoid using odd numbers for sizing.